src/main/java/resources/NasaResource.java

```diff
@@ -0,0 +1,43 @@
+package resources;
+
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.QueryParam;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import jakarta.ws.rs.core.Response.Status;
+import millom.sandbox.service.WeatherService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Path("/nasaapi")
+public class NasaResource {
+  private static final Logger LOGGER
+      = (Logger) LoggerFactory.getLogger(NasaResource.class);
```

Comment:
Is the cast needed?

Reply:
casting is removed.

`  private static final Logger LOGGER = LoggerFactory.getLogger(NasaResource.class);`